src/DistributedMemcacheMutex.php

```diff
@@ -4,6 +4,7 @@
 
 use InvalidArgumentException;
 use Memcache;
+use Memcached;
```

Comment:
Not used, at least in the changes i can see 😄

Reply:
True :)